src/use_case/search/SearchOutputData.java

```diff
@@ -5,7 +5,7 @@
 public class SearchOutputData {
     private final String stationName;
     private final String stationParentLine;
-    private final List<String> stationAmenities;
+    private final String stationAmenities; //TODO: TEMP CHANGE TO STRING TYPE
```

Comment:
Are we still planning on keeping this as a String?

Reply:
Most likely yes, for now at least